app/boards/shields/zmk_uno/zmk_uno.conf

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) 2022 The ZMK Contributors
+# SPDX-License-Identifier: MIT
```

Comment:
Missing `CONFIG_EC11=y` and global thread selection?

Reply:
Oops. Added.